prisma/schema.prisma

```diff
@@ -176,9 +177,9 @@ model Module {
 }
 
 model Exercise {
-  id          Int       @id @default(autoincrement())
-  createdAt   DateTime  @default(now()) @db.Timestamptz(6)
-  updatedAt   DateTime  @updatedAt @db.Timestamptz(6)
+  id          Int                  @id @default(autoincrement())
```

Comment:
What caused the change in formatting? The IDE?

Reply:
I ran `yarn prisma format` and `yarn prisma migrate dev`